benchmark/tool/C/inference.cc

```diff
@@ -76,8 +76,7 @@ int main(int argc, char* argv[]) {
 
   {
     Timer time("init paddle");
-    char* argv[] = {(char*)"--use_gpu=False"};
-    if (paddle_init(1, (char**)argv) != kPD_NO_ERROR) {
+    if (paddle_init(0, NULL) != kPD_NO_ERROR) {
```

Comment:
In mobile inference library, the default value of `use_gpu` is false.
And some people have some doubts about the way this input parameter, so remove the `argv` in the demo.